src/components/toggle/Toggle.tsx

```diff
@@ -18,6 +22,8 @@ export interface ToggleProps extends Omit<BoxProps, 'className' | 'size'> {
   label?: ReactNode;
   onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
   size?: Exclude<typeof SIZES[number], 'tiny' | 'fullscreen' | 'smallest' | 'hero'>;
+  tooltip?: ReactNode;
```

Comment:
Should be `string` probably. Also `tooltipLabel` would be a more suitable name I think.

Reply:
We use `ReactNode` for tooltips as there may be elements/components like links inside

Reply:
Tooltips can't have links inside of them.

Reply:
for example Company Dialog needs to have it as `ReactNode` to display `<ul>`

Reply:
Ah, HTML tags. Makes sense 👍